src/JuMP.jl

```diff
@@ -396,6 +397,37 @@ function add_bridge(model::Model,
     return
 end
 
+"""
+     print_bridge_graph(model::Model)
+
+Print out the bridge graph explaining which bridges are used (and in what order
+thay are used) to bridge relevant constraints.
```

Comment:
thay -> they

Reply:
Maybe add a note that this is the full bridge graph containing all the constraint, variable and objective types that could be obtained by bridging the variable, constraint and objective that are present in the model but this does not mean that all these bridges  and nodes are used. This is all the nodes of the full graph but then we select the shortest path and we only use this one so many nodes may be unused.

Reply:
Is there a way to print the optimal hyper-paths? That's perhaps more useful than the complete graph.

Reply:
Not yet, That would be a nice addition indeed. Other useful printing would be one showing alternative paths with the same or lower cost than the one chosen. That's the second item of https://github.com/jump-dev/MathOptInterface.jl/issues/680

Reply:
Please checkout the PR [jump-dev/MathOptInterface.jl#1055](https://github.com/jump-dev/MathOptInterface.jl/pull/1055), I think it is  solving the second item of [jump-dev/MathOptInterface.jl#680](https://github.com/jump-dev/MathOptInterface.jl/issues/680).